### ISSUE TITLE
Very strange limit on the qty

### ISSUE THREAD

**Issue by pedanticantic** [dignat/fastFood#12]
If you add an item to the cart and click Increase repeatedly, when the qty hits 45, it resets it to 0.

- There seems to be no reason for this. Why 45? Why reset the qty?
- The user is not warned that this will happen
- If there is a hard limit of 45, it should just prevent further increases. Ie stop at 45, don't reset!

**Comment by dignat** [comment]
you are right, the task had to limit the number of items that could be ordered and as I do not have DB ,can not controller the inventory , so just for test reasons i limited it to 45 and then reset, but you are right that it should stop at the limit and do not reset.
